guides/common/modules/proc_preparing-a-cloud-init-template.adoc

```diff
@@ -0,0 +1,84 @@
+[id="preparing-a-cloud-init-template_{context}"]
+= Preparing a Cloud-init Template
+
+ifeval::["{build}" == "foreman"]
+.Prerequisites
+
+* You must enable the remote execution plug-in to use a 'cloud-init' template.
+endif::[]
+
+.Procedure
+
+. In the {Project} web UI, navigate to *Hosts* > *Provisioning Templates*, and click *Create Template*.
+. In the *Name* field, enter a name for the template.
+. In the *Editor* field, enter the following template details:
++
+[options="nowrap", subs="+attributes"]
+----
+<%#
+kind: user_data
+name: My {Project} {oVirt} Cloud-init
```

Comment:
Those attributes do not work.

Reply:
Don't they? They do in my local builds. I had to remove "quotes" because it made some of the other text render weirdly when it shouldn't render at all, but the attributes should still be fine.

Reply:
My fault, I was on a wrong branch :-/